main.go

```diff
@@ -243,17 +254,17 @@ func addPodSecurityPrivilegedLabels(watchNamespaceName string) error {
 		return err
 	}
 
-	privilegedLabels := map[string]string{
-		"pod-security.kubernetes.io/enforce": "privileged",
-		"pod-security.kubernetes.io/audit":   "privileged",
-		"pod-security.kubernetes.io/warn":    "privileged",
-	}
+	namespaceLabels := operatorNamespace.GetLabels()
+	// overwrite PSA labels, if they exist; otherwise, add them
+	namespaceLabels[enforceLabel] = privileged
+	namespaceLabels[auditLabel] = privileged
+	namespaceLabels[warnLabel] = privileged
 
-	operatorNamespace.SetLabels(privilegedLabels)
+	operatorNamespace.SetLabels(namespaceLabels)
 
 	_, err = clientset.CoreV1().Namespaces().Update(context.TODO(), operatorNamespace, metav1.UpdateOptions{})
```

Comment:
Also can you switch this from update to patch?

patch call have less issues with "in-cluster version is different than ..."

Reply:
will test if patch works; if yes, will change to patch

Reply:
I would 1. test that this works by adding [envtest](https://github.com/openshift/oadp-operator/blob/20ee34398f28f9be881689f31665926ccbc2f455/pkg/credentials/credentials_test.go#L502) or [fakeClient](https://github.com/openshift/oadp-operator/blob/20ee34398f28f9be881689f31665926ccbc2f455/controllers/bsl_test.go#L40) based unitTest

Reply:
Here's an example
https://stackoverflow.com/a/65889892

Another example from @shubham-pampattiwar https://github.com/openshift/csi-external-snapshotter/commit/ff71329d8cc08dca53194f8d4568ed53e69ecb82

Reply:
I think is not possible to write unit tests to this 

I am not getting Patch to work, I tried
```go
       labelPatch := fmt.Sprintf(
		`{"metadata": {"labels": {"%s": "%s", "%s": "%s", "%s": "%s"}}}`,
		enforceLabel, privileged, auditLabel, privileged, warnLabel, privileged,
	)
	labelPatchBytes, err := json.Marshal(labelPatch)
	if err != nil {
		return err
	}

	_, err = clientset.CoreV1().Namespaces().Patch(context.TODO(), operatorNamespace.Name, types.MergePatchType, labelPatchBytes, metav1.PatchOptions{})
```

Reply:
I can add unit tests if needed. It definitely is possible.

Reply:
unit test added.

Reply:
It is now using patch.

https://github.com/openshift/oadp-operator/pull/1274#commits-pushed-74e18e8

Resolving.